packages/swapper/src/swappers/thorchain/utils/estimateTradeFee/estimateTradeFee.ts

```diff
@@ -88,21 +104,36 @@ export const estimateTradeFee = async (
   const buyFeeAsset = new AssetService().getAll()[buyFeeAssetId]
 
   switch (buyChainId) {
-    case 'bip122:000000000019d6689c085ae165831e93':
+    case KnownChainIds.BitcoinMainnet:
+      return fromBaseUnit(
+        bnOrZero(btcEstimate(gasRate)).times(buyFeeAssetRatio).dp(0),
+        buyFeeAsset.precision
+      )
+    case KnownChainIds.DogecoinMainnet:
       return fromBaseUnit(
-        bnOrZero(btcEstimate(gasRate)).times(feeAssetRatio).dp(0),
+        bnOrZero(dogeEstimate(gasRate)).times(buyFeeAssetRatio).dp(0),
         buyFeeAsset.precision
       )
-    case 'eip155:1':
+    case KnownChainIds.LitecoinMainnet:
+      return fromBaseUnit(
+        bnOrZero(ltcEstimate(gasRate)).times(buyFeeAssetRatio).dp(0),
+        buyFeeAsset.precision
+      )
+    case KnownChainIds.CosmosMainnet:
+      return fromBaseUnit(
+        bnOrZero(gaiaEstimate(gasRate)).times(buyFeeAssetRatio).dp(0),
+        8 // because thorchain likes to be inconsistent for no good reason
```

Comment:
```suggestion
        THORCHAIN_FIXED_PRECISION
```

Reply:
should this actually be `THORCHAIN_FIXED_PRECISION` everywhere above for safety - this is a protocol design thing and those other assets may happen to match but only by chance?

Reply:
its just inconsistent. It wont work for eth or erc20. But it would work for utxo because it happens to be the same value. So im not sure which to use. Minimizing THORCHAIN_FIXED_PRECISION to this one spot for now seems correct enough to me

Reply:
The eth rate comes back in terms of gwei (10^9), the btc comes back in sats (which also happens to be the thorchain precision). Atom comes back in the thochain precision.

Looking at it this way it feels like atom is the special case at least until we have more chains and want to change it up